README.md

```diff
@@ -21,7 +22,14 @@ This pod requires a deployment target of iOS 9.0 or greater
 
 ## Installation
 
-CenteredCollectionView is available through [CocoaPods](http://cocoapods.org) and [Carthage](https://github.com/Carthage/Carthage).
+CenteredCollectionView is available through [Swift Package Manager](https://swift.org/package-manager/), [CocoaPods](http://cocoapods.org) and [Carthage](https://github.com/Carthage/Carthage).
+
+To install it with **Swift Package Manager**, add the URL `https://github.com/BenEmdon/CenteredCollectionView` in Xcode Add Package Dependency assistant ; or add to your own `Package.swift`:
+```swift
+dependencies: [
+  .package(url: "https://github.com/BenEmdon/CenteredCollectionView", from: "2.2.1")
```

Comment:
I think we should release a new version for this change 👊 
```suggestion
  .package(url: "https://github.com/BenEmdon/CenteredCollectionView", from: "2.2.2")
```

Reply:
Yes we should ; but version management is up to maintainers ;-)